src/videojs-hls.js

```diff
@@ -267,9 +267,16 @@ videojs.Hls.prototype.handleSourceOpen = function() {
   // construct the video data buffer and set the appropriate MIME type
   var
     player = this.player(),
-    sourceBuffer = this.mediaSource.addSourceBuffer('video/flv; codecs="vp6,aac"');
+    mediaSource = this.mediaSource,
+    sourceBuffer = mediaSource.addSourceBuffer('video/flv; codecs="vp6,aac"');
 
   this.sourceBuffer = sourceBuffer;
+
+  // signal whenever the buffer flushes to the swf
+  this.sourceBuffer.addEventListener('updateend', function() {
+    mediaSource.endOfStream();
```

Comment:
Yes, it was.  Most of this logic has now been moved into the mediaSources project via https://github.com/videojs/videojs-contrib-media-sources/pull/24